src/vorta/application.py

```diff
@@ -194,7 +194,7 @@ def check_darwin_permissions(self):
         This function tries reading a file that is known to be restricted and warn the user about
         incomplete backups.
         """
-        test_path = Path.expanduser('~/Library/Cookies')
+        test_path = Path.expanduser(PosixPath('~/Library/Cookies'))
```

Comment:
I think this might work!

Reply:
Why? It won't help fixing the tests.

Reply:
```suggestion
        test_path = Path('~/Library/Cookies').expanduser()
```

will do the trick. `expanduser` is an instance method not a static method.